deployment/provision_vagrant_vm.sh

```diff
@@ -11,7 +11,6 @@ curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
 echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
 
 apt-get update
-apt-get -q install -y yarn
```

Comment:
why do you remove this?

Reply:
You're right, i got the wrong line, should be fixed now.